### PR TITLE
Add maintenance window option to Cloud SQL

### PR DESCRIPTION
**Pull request by mattkaem**

### CODE REVIEW

File: terraform/infrastructure/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ No requirements.
...
-| <a name="input_gke_cluster_maintenance_policy_recurring_window"></a> [gke\_cluster\_maintenance\_policy\_recurring\_window](#input\_gke\_cluster\_maintenance\_policy\_recurring\_window) | The recurring window maintenance policy for the cluster. For details see: https://registry.terraform.io/providers/hashicorp/google/5.15.0/docs/resources/container_cluster#nested_maintenance_policy | <pre>object({<br>    start_time = string,<br>    end_time = string,<br>    recurrence = string<br>  })</pre> | `null` | no |
+| <a name="input_gke_cluster_maintenance_policy_recurring_window"></a> [gke\_cluster\_maintenance\_policy\_recurring\_window](#input\_gke\_cluster\_maintenance\_policy\_recurring\_window) | The recurring window maintenance policy for the cluster. For details see: https://registry.terraform.io/providers/hashicorp/google/5.15.0/docs/resources/container_cluster#nested_maintenance_policy | <pre>object({<br>    start_time = string,<br>    end_time   = string,<br>    recurrence = string<br>  })</pre> | `null` | no |
```
Comment (julian-sotec) [inline terraform/infrastructure/README.md:37]: Why is the gke cluster maintenance policy also part of this MR? Didn't we already merge this?

File: terraform/infrastructure/variables.tf [hunk 2]
```diff
@@ -54,11 +54,11 @@ variable "gke_cluster_name" {
 variable "gke_cluster_maintenance_policy_recurring_window" {
   type = object({
     start_time = string,
-    end_time = string,
+    end_time   = string,
```
Comment (julian-sotec) [inline terraform/infrastructure/variables.tf:57]: What did change here? a single blank?

Comment (mattkaem) [reply terraform/infrastructure/variables.tf:57]: Yes, it seems like this was not formatted correctly.

File: terraform/infrastructure/README.md (hunk 1 above)
Comment (mattkaem) [reply terraform/infrastructure/README.md:37]: This is just a formatting change.

**Review by julian-sotec — APPROVED** [review]
LGTM

**Pull request merged**